<?xml version="1.0"?>
<launch>
	<arg name="hz" default="10"/>
	<arg name="epsilon" default="0.1"/>
	<arg name="max_speed" default="0.6"/>
	<arg name="max_yawrate" default="0.6"/>
	<arg name="model_path" default="/home/amsl/models/libtorch_model/ppo_model.pt"/>

	<node name="demo_navigator" pkg="motion_planner_with_drl" type="demo_navigator"/>
    <node name="mpdrl" pkg="motion_planner_with_drl" type="motion_planner_with_drl" output="screen">
		<param name="HZ" value="$(arg hz)" type="int"/>
		<param name="EPSILON" value="$(arg epsilon)" type="double"/>
		<param name="MAX_SPEED" value="$(arg max_speed)" type="double"/>
		<param name="MAX_YAWRATE" value="$(arg max_yawrate)" type="double"/>
		<param name="MODEL_PATH" value="$(arg model_path)" type="str"/>
		<remap from="/local_map/scan" to="/scan"/>
		<remap from="/local_path/cmd_vel" to="/t_frog/cmd_vel"/>
	</node>
</launch>
